docs/adr/0095-external-drive-file-picker-integration.md

```
@@ -0,0 +1,183 @@
+    "mimeType": "string",
+    "sharingLink": "string (optional)",
+    "downloadLink": "string (optional)"
+  }]
+}
+```
+
+Map each item to a `DriveAttachment` entity. Render in the existing composer attachment panel alongside regular uploaded files.
+
+On send, include each drive attachment as a JMAP `EmailBodyPart` header:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)